src/plugins/filter/filter.pop.tsx

```diff
@@ -166,25 +215,83 @@ export class FilterPanel {
   private onSave() {
     this.assertChanges();
 
+    if (!this.filterItems[this.changes.prop]) {
+      this.filterItems[this.changes.prop] = [];
+    }
+
+    this.filterItems[this.changes.prop].push({
+      id: this.filterId++,
+      type: this.changes.type,
+      value: this.extraElement?.value?.trim(),
+      relation: 'and',
+    });
+
+    this.multiFilterChange.emit(this.filterItems);
+
     this.filterChange.emit({
       prop: this.changes.prop,
       type: this.changes.type,
       value: this.extraElement?.value?.trim(),
     });
-    this.changes = undefined;
+
+    // this.changes = undefined;
```

Comment:
why did you remove changes clear?

Reply:
to not close the filter popup right away after adding a filter. very useful when adding multiple filters. just a better UX in my opinion